BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudSocialLeaderboard.cs

```diff
@@ -625,13 +625,15 @@ public enum SortOrder
         /// <param name="cbObject">
         /// The user object sent to the callback.
         /// </param>
+
+        [Obsolete("Will be removed March 2020, Please use PostScoreToDynamicLeaderboardUTC")]
         public void PostScoreToDynamicLeaderboard(
             string leaderboardId,
             long score,
             string jsonData,
             SocialLeaderboardType leaderboardType,
             RotationType rotationType,
-            DateTime? rotationReset,
+            DateTime? rotationResetUTC,
```

Comment:
i dont really know what the ? means. I don't thinks this is an optional argument.